ecosystem/sep-0012.md

```diff
@@ -107,14 +108,31 @@ Name | Type | Description
 -----|------|------------
 `id` | string | (optional) ID of the customer, if the customer has already been created via a `PUT /customer` request.
 `status` | string | Status of the customers KYC process.
-`fields` | object | (optional) An object containing the fields still needed to finish KYC Process for the given KYC type.
+`fields` | object | (optional) An object containing the fields for the given KYC type. See [fields](#fields) for more detailed information.
```

Comment:
❓ The fields used to contain the fields still needed, but now it sounds like it will contain all the fields. It sounds like the user could provide the fields in multiple calls before. Now must it be in one call? Or was it always intended to be a single call?

Reply:
It was never clearly defined whether multiple calls were acceptable or single calls required. Polaris leaves this decision up to the anchor.

I don't see a reason to pick a policy here either. In fact, doing so would be a breaking change. Anchors can return `400` if they'd like all fields at once, and its likely less complex to implement that way.

However, I do see the possibility of the client wanting to verify one field at a time. From a UX perspective, the client may want to know if the value provided by the user is valid before moving on to collect other pieces of information.

If the client wants this UX, it can reach out to the anchor directly to make their implementation more flexible. The `NOT_PROVIDED` field status will be useful in this case too.